### PR TITLE
Fix/form submit buttons

### PR DESCRIPTION
**Pull request by Guillaume-FAURE**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -29,19 +29,19 @@
     <div class="button-container">
       <button id="restart">Nouvelle partie</button>
       <div class="verify-container">
-        <button id="verify">Vérifier</button>
+        <button id="verify" type="submit">Vérifier</button>
```
Comment (RomainGuarinoni) [inline public/index.html:32]: Actually this does not work because the button is not in a `<form> `

File: src/index.ts [hunk 2]
```diff
@@ -309,8 +306,11 @@ verifyButton.onclick = verifyCurrentCombination;
 
 restartButton.onclick = startNewGame;
 
-// add apply parameters event
-applyParametersButton.onclick = applyParameters;
+// add apply parameters event, usable with enter key to submit the form
+parametersPopup.addEventListener('submit', (e) => {
+  e.preventDefault();
+  applyParameters();
+});
```
Comment (RomainGuarinoni) [inline src/index.ts:313]: Can you move this listeners in the `listeners.ts `file ? Could be great to test it when we implement test ;) mb smthing like this : 

```ts
// listeners.ts

export addFormEvent(form:HTMLFormElement, action : ()=>void) { 
// implement code here
}

```

```ts
// index.ts
addFormEvent(parametersPopup,applyParameters)

**Review by RomainGuarinoni — CHANGES_REQUESTED** [review]
LGTM, just few things to change for code architecture but it works well !

File: src/index.ts (hunk 2 above)
Comment (Guillaume-FAURE) [reply src/index.ts:312]: done

File: public/index.html (hunk 1 above)
Comment (Guillaume-FAURE) [reply public/index.html:32]: removed

File: src/index.ts (hunk 2 above)
Comment (RomainGuarinoni) [reply src/index.ts:312]: Dont see the change ...

**Review by RomainGuarinoni — APPROVED** [review]
LGTM

**Pull request merged**